samples/wifi/radio_test/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+config INCLUDE_NET_CORE_IMAGE
+	bool "Include net image as sub image"
+	depends on SOC_NRF5340_CPUAPP
+	default y if (BOARD_NRF7002DK_NRF5340_CPUAPP || BOARD_NRF7002DK_NRF5340_CPUAPPNS)
```

Comment:
Board specific default here. Why not default y based on the SOC instead, so customer boards get this default too?

Reply:
As this is a sample, customer can choose to make this default or not.

Reply:
Are you saying that this is not a good default for all use cases on this SoC?

Reply:
Including CPUNET image makes sense only if this sample is compiled for CPUAPP independent of SoC (after 5340 which as of now is 5340 and 7002 boards). AFAIK, we don't have a CPUAPP Kconfig without being tied to a board, do we ?